### ISSUE TITLE
1.5.0 RC1 Tandem XE switch mapping - checklist becomes incomprehensible with incorrect switch positions

### ISSUE THREAD

**Issue by yak55** [FrSkyRC/ETHOS-Feedback-Community#3365]
1.5.0 RC1 Tandem XE switch mapping - checklist becomes incomprehensible with incorrect switch positions
(Tested on the transmitter - not on the simulator)

In my example, I have mapped switch #2 (SB) to switch 7.
![IMG_6311](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/85737356/8c629583-960b-4002-a2f1-00c67806cf28)

If **switch #2 is set incorrectly** when switching on, the following display appears:

![IMG_6312](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/85737356/b215dfd5-3d47-4680-bdf6-a779d4146947)

If you have not memorized the mapped switches, there will be many attempts until the switches are set correctly and the transmitter switches on.

Greetings Eugen

**Comment by hrenz** [comment]
Schaltermitte sollte man 150ms eintragen, damit beim schnellen umschalten nicht dauernd noch die Mitte mit erkannt werden, die man da gar nicht will.
0ms macht keinen Sinn, damit wird falsch ausgewertet.
Mach es einfach mal und du wirst sehen es geht.

Das sollte schon in der Voreinstellung 150ms stehen oder hast du das geändert?

**Comment by yak55** [comment]
Hello Helmut, thank you for your advice.

As I expected, it did NOT lead to success.
The problem already arises when the transmitter is switched on, without the switches being moved.

It is the result of a switch being mapped on the XE.
The original switch #2 is wrong, but the mapping target #7 is shown in the graphic.

Here we have to wait for Bertrand.

About the rest: I have been using an X20, an XE and an X20 Pro since they became available.
With all 3 FST. The value is set to 0 milliseconds and I've never had the slightest problem with it.

Have a nice weekend

Greetings Eugen

**Comment by bsongis-frsky** [comment]
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/1348e5fa-718f-4bf3-a75c-80e543b47bea)
 I am not sure I understand correctly the issue, the names displayed are not correct? The names you have entered in System / Hardware are not easy to follow!

**Comment by yak55** [comment]
Hello Bertrand,there is the following problem with the XE:
In the photo I have mapped switch 2 SB to switch 7.
The function is ok.
But: if the physical switch 2 is wrong when the remote control is switched on, the pictogram for the mapped switch 7 is displayed in the checklist.
So you would now change switch 7.
The pictogram says yes, it is incorrect.
But it doesn't help, because the cause is switch 2.
This is very confusing.
The pictogram should be on 2, not 7.
Tested on the XE remote control.
Also affects the other mapped switches.

Greetings Eugen
